airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java

```diff
@@ -59,18 +65,18 @@ public void filter(final ContainerRequestContext requestContext) throws IOExcept
       requestContext.setEntityStream(new ByteArrayInputStream(baos.toByteArray()));
       // end hack
 
-      requestBody = IOUtils.toString(entity, MessageUtils.getCharset(requestContext.getMediaType()));
+      MDC.put(MDC_RESPONSE_BODY_KEY, IOUtils.toString(entity, MessageUtils.getCharset(requestContext.getMediaType())));
     }
   }
 
   @Override
   public void filter(final ContainerRequestContext requestContext, final ContainerResponseContext responseContext) {
-    MDC.setContextMap(mdc);
```

Comment:
@jrhizor when you're back from PTO, can you check on this and let me know if you foresee any issue with moving the `MDC.setContextMap` call from this response filter to the request filter above?